### ISSUE TITLE
启动web demo.无法访问指定的端口，报错404

### ISSUE THREAD

**Issue by chengzhen123** [FunAudioLLM/Fun-Audio-Chat#15]
### 启动脚本是：

`python -m web_demo.server.server --model-path pretrained_models/Fun-Audio-Chat-8B --port 10095 --tts-gpu 2`

在容器中启动的指定的脚本，容器的端口映射是 -p 12095:10095

### 启动信息如下：

`[INFO] Initializing server with Opus sample rate: 24000, Model sample rate: 16000
[INFO] loading s2s model to cuda...
Loading checkpoint shards: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:00<00:00, 48.32it/s]
[INFO] s2s model loaded (: cuda)
[INFO] waiting for tts model loading ... 
[INFO] Output directory: ./output
[INFO] Global TTS process started (pid: 4490)
[INFO] Access the Web UI directly at http://localhost:10095
2025-12-26 14:12:47,498 DEBUG Using selector: EpollSelector
======== Running on http://localhost:10095 ========
(Press CTRL+C to quit)
[INFO] [TTS Process] Starting TTS worker process on cuda:2...
[INFO] [TTS Process] Loading TTS model...
2025-12-26 14:12:53.409 | INFO     | utils.cosyvoice_detokenizer:get_audio_detokenizer:87 - cosyvoice cuda: True
/root/anaconda3/envs/FunAudioChat/lib/python3.12/site-packages/lightning/fabric/__init__.py:41: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html. The pkg_resources package is slated for removal as early as 2025-11-30. Refrain from using this package or pin to Setuptools<81.
2025-12-26 14:12:56,299 INFO input frame rate=25
/root/anaconda3/envs/FunAudioChat/lib/python3.12/site-packages/torch/nn/utils/weight_norm.py:144: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
2025-12-26 14:12:58.313457923 [E:onnxruntime:Default, provider_bridge_ort.cc:1744 TryGetProviderInfo_CUDA] /onnxruntime_src/onnxruntime/core/session/provider_bridge_ort.cc:1426 onnxruntime::Provider& onnxruntime::ProviderLibrary::Get() [ONNXRuntimeError] : 1 : FAIL : Failed to load library libonnxruntime_providers_cuda.so with error: libcudnn.so.8: cannot open shared object file: No such file or directory

2025-12-26 14:12:58.313487193 [W:onnxruntime:Default, onnxruntime_pybind_state.cc:870 CreateExecutionProviderInstance] Failed to create CUDAExecutionProvider. Please reference https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirementsto ensure all dependencies are met.
2025-12-26 14:12:59,227 DEBUG Attempting to acquire lock 140419425067440 on /root/.cache/modelscope/hub/.lock/pengzhendong___wetext
2025-12-26 14:12:59,227 DEBUG Lock 140419425067440 acquired on /root/.cache/modelscope/hub/.lock/pengzhendong___wetext
2025-12-26 14:12:59,231 DEBUG Starting new HTTPS connection (1): www.modelscope.cn:443
2025-12-26 14:12:59,391 DEBUG https://www.modelscope.cn:443 "GET /api/v1/repos/internalAccelerationInfo HTTP/1.1" 200 188
2025-12-26 14:12:59,393 DEBUG Starting new HTTP connection (1): 100.100.100.200:80
2025-12-26 14:12:59,595 DEBUG Starting new HTTPS connection (1): www.modelscope.cn:443
2025-12-26 14:12:59,786 DEBUG https://www.modelscope.cn:443 "GET /api/v1/models/pengzhendong/wetext HTTP/1.1" 200 None
Downloading Model from https://www.modelscope.cn to directory: /root/.cache/modelscope/hub/models/pengzhendong/wetext
2025-12-26 14:13:00,009 DEBUG https://www.modelscope.cn:443 "GET /api/v1/models/pengzhendong/wetext/revisions HTTP/1.1" 200 222
2025-12-26 14:13:00,116 DEBUG https://www.modelscope.cn:443 "GET /api/v1/models/pengzhendong/wetext/repo/files?Revision=master&Recursive=True HTTP/1.1" 200 None
2025-12-26 14:13:00,118 DEBUG Attempting to release lock 140419425067440 on /root/.cache/modelscope/hub/.lock/pengzhendong___wetext
2025-12-26 14:13:00,118 DEBUG Lock 140419425067440 released on /root/.cache/modelscope/hub/.lock/pengzhendong___wetext
2025-12-26 14:13:00,235 DEBUG Attempting to acquire lock 140407709154864 on /root/.cache/modelscope/hub/.lock/pengzhendong___wetext
2025-12-26 14:13:00,236 DEBUG Lock 140407709154864 acquired on /root/.cache/modelscope/hub/.lock/pengzhendong___wetext
2025-12-26 14:13:00,237 DEBUG Starting new HTTPS connection (1): www.modelscope.cn:443
2025-12-26 14:13:00,398 DEBUG https://www.modelscope.cn:443 "GET /api/v1/repos/internalAccelerationInfo HTTP/1.1" 200 188
2025-12-26 14:13:00,400 DEBUG Starting new HTTP connection (1): 100.100.100.200:80
2025-12-26 14:13:00,601 DEBUG Starting new HTTPS connection (1): www.modelscope.cn:443
2025-12-26 14:13:00,822 DEBUG https://www.modelscope.cn:443 "GET /api/v1/models/pengzhendong/wetext HTTP/1.1" 200 None
Downloading Model from https://www.modelscope.cn to directory: /root/.cache/modelscope/hub/models/pengzhendong/wetext
2025-12-26 14:13:00,938 DEBUG https://www.modelscope.cn:443 "GET /api/v1/models/pengzhendong/wetext/revisions HTTP/1.1" 200 222
2025-12-26 14:13:01,046 DEBUG https://www.modelscope.cn:443 "GET /api/v1/models/pengzhendong/wetext/repo/files?Revision=master&Recursive=True HTTP/1.1" 200 None
2025-12-26 14:13:01,048 DEBUG Attempting to release lock 140407709154864 on /root/.cache/modelscope/hub/.lock/pengzhendong___wetext
2025-12-26 14:13:01,048 DEBUG Lock 140407709154864 released on /root/.cache/modelscope/hub/.lock/pengzhendong___wetext
2025-12-26 14:13:01,195 INFO use wetext frontend
2025-12-26 14:13:03.445 | INFO     | utils.cosyvoice_detokenizer:get_audio_detokenizer:91 - cosyvoice loaded
[INFO] [TTS Process] TTS model loaded successfully
`

### 报错信息如下：

在网页无法访问 http://localhost:12095

访问 http://localhost:12095/api/chat 显示 
`No WebSocket UPGRADE hdr: None
 Can "Upgrade" only to "WebSocket"`

**Comment by Junehom** [comment]
这是因为这个后端服务的启动脚本是没有前端，要通过网页访问可以通过--static配置一个自己另外写好的前端或者用我们给的wedemo.client 启动前端。
后端服务启动之后，在client的 .env.local里面将VITE_SIMPLEX_TARGET配置成你启动好的后端服务http://localhost:12095/ 网页通过前端端口访问。启动方式参考主目录下面readme的client部分
